src/lib/redux/inventoryDataSlice.ts

```diff
@@ -48,6 +53,24 @@ export const EMPTY_INVENTORY_UPDATE: InventoryUpdateRecord = {
   period: 0,
 };
 
+export enum PurchaseRequestStatus {
+  Approved = "Approved",
+  Denied = "Denied",
```

Comment:
Lets follow the all-caps enum convention we have in codebase already https://github.com/calblueprint/meepanyar/blob/main/src/lib/i18next/translator.ts#L8